ctapipe/atmosphere.py

```diff
@@ -169,7 +169,7 @@ class ExponentialAtmosphereDensityProfile(AtmosphereDensityProfile):
         density_profile.peek()
 
 
-    Attributes
+    Parameters
```

Comment:
But attributes is correct. It's about the class attributes. Parameters are function arguments

Reply:
you're totally right

Reply:
don't know why sphinx didn't want to build it on my machine

Reply:
I'll close this again